lstchain/scripts/lstchain_data_muon_analysis_dl1.py

```diff
@@ -13,6 +13,7 @@
 
 from lstchain.image.muon import analyze_muon_event, muon_filter, tag_pix_thr
 from lstchain.io.io import dl1_params_lstcam_key, dl1_images_lstcam_key
+from lstchain.visualization import plot_calib as calib
```

Comment:
Since there is a module called `calib` in lstchain, I would recommend removing the `as calib` part and use `plot_calib` as is to avoid confusion

Reply:
Done